distro/debian/patches/series

```diff
@@ -1,4 +1,2 @@
 tests-disable-coupling_tools.patch
 discard-levelsetmesher-test.patch
-fix-multiprocessing-hurd.patch
-test-NormalGamma.patch
```

Comment:
I will have to keep test-NormalGamma.patch in Debian

Reply:
Why is that ? I applied it.